### PR TITLE
Add auto roller to GitHub Actions.

### PR DESCRIPTION
**Pull request by jianjunz**
Please merge this PR after https://github.com/open-webrtc-toolkit/owt-client-native/pull/506.

### CODE REVIEW

**Comment by jianjunz** [comment]
@taste1981, please take a look.  open-webrtc-toolkit/owt-client-native#506 is merged.

File: .github/workflows/roll.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      with:
+        repository: open-webrtc-toolkit/owt-client-native
+        ref: refs/heads/master
+        fetch-depth: 0
+        token: ${{ secrets.OWTBOT_TOKEN }}
+    - uses: actions/setup-python@v2
+    - run: pip install requests
+    - run: python scripts/roll_webrtc.py --base_branch master --revision $GITHUB_SHA --token $OWTBOT_TOKEN
+      env:
+        OWTBOT_TOKEN: ${{ secrets.OWTBOT_TOKEN }}
```
Comment (taste1981) [inline .github/workflows/roll.yaml]: OWT_BOT_TOKEN instead of OWTBOT_TOKEN....

File: .github/workflows/roll.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Roll WebRTC revision
```
Comment (taste1981) [inline .github/workflows/roll.yaml:1]: I assume you mean "version" instead of "revision".

Comment (jianjunz) [reply .github/workflows/roll.yaml:1]: Revision should be okay here?

File: .github/workflows/roll.yaml (hunk 1 above)
Comment (jianjunz) [reply .github/workflows/roll.yaml]: Done.

**Pull request merged**